app/models/merchant.rb

```diff
@@ -28,6 +28,28 @@ def self.with_most_items(quantity)
       .limit(quantity)
   end
 
+  def customers_with_pending_invoices
+    Customer.find_by_sql("select customers.* from customers
+      join invoices
+      on customers.id = invoices.customer_id
+      where invoices.id in (
+        select invoices.id
+        from invoices
+        join transactions
+        on invoices.id = transactions.invoice_id
```

Comment:
I have a feeling we could extract this subquery to invoices, if we ever want to.